pkg/planner/core/casetest/testdata/integration_suite_out.json

```diff
@@ -1384,5 +1384,284 @@
+          "  └─TableFullScan_8 16384.00 3576605.25 ((lm_col_scan(42768*logrowsize(16)*tiflash_scan_factor(11.6))) + (lm_rest_col_scan(16384*logrowsize(48)*tiflash_scan_factor(11.6)*lm_scan_factor(1.5))))*1.00 mpp[tiflash] table:t1 pushed down filter:gt(test.t1.a, 2), keep order:false"
+        ]
+      },
+      {
+        "SQL": "explain format='cost_trace' select * from t1 where a=1",
+        "Plan": [
+          "TableReader_10 8192.00 223826.29 (((((((lm_col_scan(42768*logrowsize(16)*tiflash_scan_factor(11.6))) + (lm_rest_col_scan(8192*logrowsize(48)*tiflash_scan_factor(11.6)*lm_scan_factor(1.5))))*1.00) + ((cpu(8192*filters(0.04)*tiflash_cpu_factor(2.4)))/5.00))) + (net(8192*rowsize(32)*tidb_flash_net_factor(2.2))))/15.00)*1.00 root  MppVersion: 3, data:ExchangeSender_9",
+          "└─ExchangeSender_9 8192.00 2780677.51 ((((lm_col_scan(42768*logrowsize(16)*tiflash_scan_factor(11.6))) + (lm_rest_col_scan(8192*logrowsize(48)*tiflash_scan_factor(11.6)*lm_scan_factor(1.5))))*1.00) + ((cpu(8192*filters(0.04)*tiflash_cpu_factor(2.4)))/5.00)) mpp[tiflash]  ExchangeType: PassThrough",
+          "  └─TableFullScan_8 8192.00 2780520.22 ((lm_col_scan(42768*logrowsize(16)*tiflash_scan_factor(11.6))) + (lm_rest_col_scan(8192*logrowsize(48)*tiflash_scan_factor(11.6)*lm_scan_factor(1.5))))*1.00 mpp[tiflash] table:t1 pushed down filter:eq(test.t1.a, 1), keep order:false"
+        ]
+      },
+      {
+        "SQL": "explain format='cost_trace' select * from t1 where a in (1,2,3)",
+        "Plan": [
+          "TableReader_12 24576.00 319061.22 ((((((cpu(32768*filters(1)*tiflash_cpu_factor(2.4))) + (((scan(32768*logrowsize(64)*tiflash_scan_factor(11.6))) + (scan(10000*logrowsize(64)*tiflash_scan_factor(11.6))))*1.00)) + ((cpu(24576*filters(0.04)*tiflash_cpu_factor(2.4)))/5.00))) + (net(24576*rowsize(32)*tidb_flash_net_factor(2.2))))/15.00)*1.00 root  MppVersion: 3, data:ExchangeSender_11",
+          "└─ExchangeSender_11 24576.00 3055767.86 (((cpu(32768*filters(1)*tiflash_cpu_factor(2.4))) + (((scan(32768*logrowsize(64)*tiflash_scan_factor(11.6))) + (scan(10000*logrowsize(64)*tiflash_scan_factor(11.6))))*1.00)) + ((cpu(24576*filters(0.04)*tiflash_cpu_factor(2.4)))/5.00)) mpp[tiflash]  ExchangeType: PassThrough",
+          "  └─Selection_10 24576.00 3055296.00 (cpu(32768*filters(1)*tiflash_cpu_factor(2.4))) + (((scan(32768*logrowsize(64)*tiflash_scan_factor(11.6))) + (scan(10000*logrowsize(64)*tiflash_scan_factor(11.6))))*1.00) mpp[tiflash]  in(test.t1.a, 1, 2, 3)",
+          "    └─TableFullScan_9 32768.00 2976652.80 ((scan(32768*logrowsize(64)*tiflash_scan_factor(11.6))) + (scan(10000*logrowsize(64)*tiflash_scan_factor(11.6))))*1.00 mpp[tiflash] table:t1 keep order:false"
```

Comment:
In the master, ```TableFullScan```'s line without PushDownFilter in the cost should be the same line. but Why do this test case has two difference row count ?

https://github.com/pingcap/tidb/blob/eeccd2e94c23df353ba70919691d65d38ad31805/pkg/planner/core/casetest/tpch/testdata/tpch_suite_out.json#L326-L327